application/models/property_builder.py

```diff
@@ -0,0 +1,46 @@
+"""Cleans up Properties for Artifact creation."""
+
+
+class PropertyBuilder:
+    """Describe a PropertyBuilder."""
+
+    def __init__(self, **properties):
+        self.__properties = properties
+        self.__relationship_properties = None
+        self.__node_properties = None
+        self.relationship_property_types = ['tags', 'user_tags', 'label_tags', 'text_tags',
+                                            'team_id']
+        self.node_property_types = ['file_url', 'file_date']

```

Comment:
Can we have these two instance variables at the top as constants? Just as capslock class variable for example.

Reply:
done